ChangeLog

```diff
@@ -17,6 +17,7 @@ version 0.24-dev0
   * Added MOVTVPRI optimization, which extends MOVTV for recording-time nil, false and true values
   * Extended information about contributors
   * Excluded movtv and movtvpri from -O4
+  * Backported fix for string.find recording from tarantool/luajit repo
```

Comment:
The fact that this is a backport is mentioned in a commit. Here, it would be more useful to describe what about string.find recording was fixed.

Reply:
IMHO, it's a bit excessive condering a conversation within the related [LuaJIT issue](https://github.com/LuaJIT/LuaJIT/issues/505).

Reply:
The line in ChangeLog doesn't point to an issue, though

Reply:
Commit message does. The issue can be obtained by blaming exactly this ChangeLog line.

Reply:
Then it's enough to write "Fix string.find fold machinery"

Reply:
No, it's a backporting of the specified commits. Yes, they are adjusted considering uJIT specific, but the original commits are from tarantool/luajit repo. The similar we have [here](https://github.com/iponweb/luavela/commit/ca0b3e2bcad72ceb71455fbff4d17f772308fd94). See no sense in the further conversation. Your move.

Reply:
The commit hash is not specified in ChangeLog, which is why I commented on that. So, for message to be helpful, it should either have a commit hash in it, or shortly describe what was fixed.